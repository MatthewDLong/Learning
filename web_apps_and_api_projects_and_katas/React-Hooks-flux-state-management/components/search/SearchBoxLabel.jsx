import React from "react";
import { searchbox__label } from "./search.css";

const SearchBoxLabel = () => (
  <label
    id="searchbox__label"
    htmlFor="searchbox__input"
    className={searchbox__label}
  >
    Pick-up Location
  </label>
);

export default SearchBoxLabel;
