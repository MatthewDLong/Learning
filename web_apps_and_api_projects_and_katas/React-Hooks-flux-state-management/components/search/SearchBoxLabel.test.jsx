import React from "react";
import SearchBoxLabel from "./SearchBoxLabel";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<SearchBoxLabel />).toJSON();
  expect(tree).toMatchSnapshot();
});
