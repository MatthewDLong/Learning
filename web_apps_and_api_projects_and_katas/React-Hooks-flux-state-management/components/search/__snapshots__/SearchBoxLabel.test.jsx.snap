// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<label
  className="searchbox__label"
  htmlFor="searchbox__input"
  id="searchbox__label"
>
  Pick-up Location
</label>
`;
